cmd/operator_install.go

```diff
@@ -33,8 +33,12 @@ func newOperatorInstallCmd(operatorConfig *operatorCommandConfig) *cobra.Command
 	// installCmd represents the "appsody deploy install" command
 	var installCmd = &cobra.Command{
 		Use:   "install",
-		Short: "Install the Appsody Operator into the configured Kubernetes cluster",
-		Long:  ``,
+		Short: "Install the Appsody Operator.",
+		Long: `Install the Appsody Operator into your configured Kubernetes cluster.
+		
+The Appsody Operator deploys an Operator container with the controller, and listens for incoming AppsodyApplication resources.`,
```

Comment:
This line doesn't sound quite right. Maybe try something like:
`The Appsody Operator listens for incoming AppsodyApplication resources on your cluster. For more information, see https://operatorhub.io/operator/appsody-operator`

Reply:
We should also clarify that unless the `--watch-all` flag us used, the operator only watches one namespace, so additional operators must be installed to watch additional namespaces.